monkey/monkey_island/cc/services/run_local_monkey.py

```diff
@@ -59,7 +65,7 @@ def run_local_monkey(self):
 
         # run the monkey
         try:
-            ip = local_ip_addresses()[0]
+            ip = self._local_ips[0]
```

Comment:
Can there be 0 ip's?

Reply:
If there are no interfaces with AF_INET addresses... doesn't seem likely.